docs/UserGuide.adoc

```diff
@@ -545,6 +545,23 @@ Example:
 * `done icP/S7896432A icD/S4578902G res/fail`
 // end::mark[]
 
+// tag::delete[]
+=== Deleting a person : `delete`
+
+If you want to delete a person in ORGANice you can type in a command in the following format: `delete **NRIC**`  +
+ORGANice will prevent you from deleting persons in the following categories:
+
+* They are a patient and their status is currently under processing.
+* They are a donor and their status is currently under processing.
+* They are a doctor and they are in charge of some number of patients in ORGANice.
+
+If you want to delete a patient that has NRIC of S4612296H and is not currently under processing, type the following command: `delete S4612296H`
```

Comment:
Here is an example. If you want to delete a patient with the NRIC S4612296H and is not under processing, type the following command: `delete S4612296H`.